takwimu/context_processors.py

```diff
@@ -152,3 +155,36 @@ def sdgs(request):
     return {
         'sdgs': sdgs,
     }
+
+
+def asset_manifest(request):
+    manifest_filepath = os.path.join(settings.BASE_DIR, 'takwimu/takwimu_ui/build/asset-manifest.json')
+    asset_manifest = {}
+
+    if os.environ.get('DEBUG', 'True') == 'True':
+        asset_manifest = {
+            'debug': True,
+            'main': 'js/main.chunk.js',
+            'vendor': 'js/bundle.js',
+            'runtime': 'js/0.chunk.js',
```

Comment:
`js/bundle.js` is the `runtime` and `js/0.chuck.js` is the `vendor`. `runtime` comes before `vendor`.